libs/block-editor/src/lib/extensions/blocks/image-block/image-block.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { AngularNodeViewComponent } from '../../../NodeViewRenderer';
+
+@Component({
+    selector: 'dotcms-image-block',
+    templateUrl: './image-block.component.html',
+    styleUrls: ['./image-block.component.scss'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
Why?

Reply:
initially because of the `img` tag, but no reason, removed.